### ISSUE TITLE
Provide the ability to disable motion blur via a setting

### ISSUE THREAD

**Issue by Nammi-namm** [Yellow-Dog-Man/Resonite-Issues#178]
### Is your feature request related to a problem? Please describe.

Many people have complained about motion blur being forced on in desktop mode. They would like to be able to turn it off.
To some it is headache inducing, especially at lower framerates.

### Describe the solution you'd like

- Having a toggle for motion blur in the settings panel. With it set to disabled by default.

### Describe alternatives you've considered

- Having motion blur able to be disabled via a launch command.

- Having motion blur off completely.

### Additional Context

There is a mod for disabling motion blur https://github.com/Raidriar796/MotionBlurDisable

**Comment by Rellik66** [comment]
This was brought up during today's office hours, TLDR; the settings need to be reworked before this will be considered.

But as a stopgap, might I propose a command line option?

**Comment by Enverex** [comment]
I've been told the team don't want to touch the Settings page due to issues with it, so why not a launch option in the same way we can toggle other features, e.g. CTAA via the -CTAA flag instead for now. For example there could be a -NoBlur switch to disable motion blur.

**Comment by ProbablePrime** [comment]
To add on from office hours, there's a bunch of problems in place here:
1. The Settings panel is bursting at the seams. Adding settings is difficult and prone to errors
2. The settings system(how they're stored, saved, loaded, recalled and accessed) needs an overhaul, Adding settings is difficult and prone to errors
3. We need to implement it in a way that makes sense for our rendering path going forwards.

From that we usually get:
- Just use Command Flags
- What about Config.json.

To that:
1. The command line flag system needs an overhaul, it is difficult to manage and add settings
2. The config system needs an overhaul, it is difficult to manage and add settings

At least we now have a self-contained, single response to most common questions regarding it. This way Rumors and misinformation can be combatted.

**Comment by XDelta** [comment]
In the meantime, a mod exists to turn off motionblur https://github.com/Raidriar796/MotionBlurDisable

**Comment by troyBORG** [comment]
Just to throw it in there.  But what about doing something like the Settings Redux mod.
https://github.com/XDelta/SettingsRedux

That way instead of saying "its at capacity" split it out into different groups so you have room to add stuff for the time being until you get an actually overhaul of the settings menu completed..

**Comment by CakePost** [comment]
> the company has made the decision to leave this "on" as in some cases this can make items look better. We understand that some of you may disagree but that's the decision that has been made.

I would just like to put my name on the list as someone who disagrees with this decision. That's all.

**Comment by ProbablePrime** [comment]
> Just to throw it in there. But what about doing something like the Settings Redux mod. [XDelta/SettingsRedux](https://github.com/XDelta/SettingsRedux)
> 
> That way instead of saying "its at capacity" split it out into different groups so you have room to add stuff for the time being until you get an actually overhaul of the settings menu completed..

Doing this would qualify as a "settings re-work" which is the work required in Item 1 of my response. Additionally the settings re-work should be a separate issue.

Once again we're not "not" doing it because we do not know how, we're not doing it because of other matters, other priorities or the appropriate work requiring more thought.

**Comment by Meister1593** [comment]
What about command line setting that other people have already done?
It does not seem to be a massive hassle to implement it as some people have demonstrated in the past.
And it will satisfy pretty much all the needs for the community in the mean time while it's being properly developed.

The way i see it myself as this is being treated as something unimportant right now, while it's in fact a massive burden on community. 
And responding with "some items look better with it" doesn't make any sense, at least to me, where motion blur could be genuinely nauseating for some people.

**Comment by CakePost** [comment]
> To that end, the company has made the decision to leave this "on" as in some cases this can make items look better.

I'm genuinely curious what items look better. Having insight into this might help alleviate some of the confusion. Some examples here would go a long way to better direct the conversation.

**Comment by ProbablePrime** [comment]
> What about command line setting that other people have already done? It does not seem to be a massive hassle to implement it as some people have demonstrated in the past. And it will satisfy pretty much all the needs for the community in the mean time while it's being properly developed.
> 
> The way i see it myself as this is being treated as something unimportant right now, while it's in fact a massive burden on community. And responding with "some items look better with it" doesn't make any sense, at least to me, where motion blur could be genuinely nauseating for some people.

A command flag is covered in a previous response, thank you.

**Comment by FlameSoulis** [comment]
>And responding with "some items look better with it" doesn't make any sense, at least to me, where motion blur could be genuinely nauseating for some people.

Bumping this because this is becoming more and more common in the Steam Discussions. This is slowly becoming less of a tolerable issue, and I'm kind of surprised this is still an issue given how much of a complaint this was in NeosVR.

**Comment by Frooxius** [comment]
We'll tackle this as part of this: https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/584

I think prioritizing the setting UI rework soon is better use of our time and energy at this point. This is a particularly noticeable issue, but there's lots of others as well and number of things are made more difficult by not having a proper system in place.

**Comment by Frooxius** [comment]
With #584 this is now unblocked :3 Just saying!

**Comment by Frooxius** [comment]
This is now implemented in 2024.4.17.1420 along with a few things! IT HAS COME! AAAAAA!
